src/ngraph/pass/constant_folding_gather.cpp

```diff
@@ -193,3 +194,62 @@ void pass::ConstantFolding::construct_constant_gather()
     this->add_matcher(
         gather_matcher_v1, constant_gather_callback, PassProperty::CHANGE_DYNAMIC_STATE);
 }
+
+void pass::ConstantFolding::construct_constant_gather_with_subgraph()
+{
+    auto concat_label = make_shared<pattern::op::Label>(
+        element::f32, Shape{2, 3, 4}, pattern::has_class<op::Concat>());
+    auto indices_label =
+        make_shared<pattern::op::Label>(element::i64, Shape{5}, pattern::has_class<op::Constant>());
+    auto axis_label =
+        make_shared<pattern::op::Label>(element::i64, Shape{1}, pattern::has_class<op::Constant>());
+    auto gather_v1 = make_shared<op::v1::Gather>(concat_label, indices_label, axis_label);
+
+    auto concat_gather_callback = [concat_label, indices_label, axis_label](pattern::Matcher& m) {
+        NGRAPH_DEBUG << "In callback for construct_constant_gather_with_subgraph against node = "
+                     << m.get_match_root()->get_name();
```

Comment:
Skip the `->get_name()` and you'll get much more useful information.

Reply:
done